### ISSUE TITLE
The HTML, which is created by the `Export vault to HTML` command has dead links

### ISSUE THREAD

**Issue by tadashi-aikawa** [KosmosisDire/obsidian-webpage-export#9]
Hi. First, thank you for developing the amazing plugin :)

I found the strange behavior about `Export vault to HTML` command. So let me inform you of the details.

----

## Version

1.3.2

## Settings

All settings are as default.

## Reproduction steps

1. Create a new vault named `webpage-test`
2. Install this plugin
3. Create a `dir` directory, and create two files located as follows

```
./webpage-test
├── 📁 dir
│   └── file in the dir.md
└── file.md
```

`file.md`
```markdown
[[file in the dir]]
```

4. Run the `Export vault to HTML` command
5. Open `file.html` in the browser

## Expected

The link URL is `./dir/file%20in%20the%20dir.html`

## Actual

The link URL is `./file%20in%20the%20dir.html`

![image](https://user-images.githubusercontent.com/9500018/211194658-25e8e841-ab4f-4ed1-b05f-201228445652.png)

----

Kind regards :)

**Comment by KosmosisDire** [comment]
This is a known issue, I believe that if you were to replace `[[file in the dir]]` with `[[dir/file in the dir]]` it would then work as expected both in obsidian and after export. However, it's definitely on my list of things to fix!

**Comment by tadashi-aikawa** [comment]
Thank you for your quick reply :)

> However, it's definitely on my list of things to fix!
Cool! 

FYR:

![image](https://user-images.githubusercontent.com/9500018/211196474-90616590-bc40-4e8a-bb3e-6ab888d62b3d.png)

**Comment by KosmosisDire** [comment]
Hey, that's awesome! Thank you! I'll actually try and add it right now :)

**Comment by KosmosisDire** [comment]
This should be fixed in the latest release 1.3.3, let me know if you find anything that doesn't work!

**Comment by tadashi-aikawa** [comment]
Thank you for your quick release! I have confirmed it, and it works perfectly 👍
